src/utils/test-utils.js

```diff
@@ -27,7 +27,7 @@ export const queryAllByTestId = queryHelpers.queryAllByAttribute.bind(
 export function getAllByTestId(container, id, ...rest) {
   const els = queryAllByTestId(container, id, ...rest);
   if (!els.length) {
-    throw queryHelpers.getElementError(
+    throw queryHelpers.getMultipleElementsFoundError(
```

Comment:
the queryHelpers object did not have getElementError as an option for a function, and I didn't find anything in the docs about it. But this getMultipleElementsFoundError was a function in the queryHelpers obj.